components/Data.vue

```diff
@@ -325,12 +288,12 @@ export default {
         filename = this.geojsonFiles[this.geojsonIndex].filename
       }
 
-      const res = await axios.get(urlString, {})
+      const res = await downloadFile(urlString)
```

Comment:
Maybe a bit confusing to have a custom `downloadFile` method when there's a `fileDownload` method already? Could change `downloadFile` to `getFileFromMinio` or something?